meeting-assistant/src/ActionHandlers.js

```diff
@@ -54,11 +76,20 @@ var ActionHandlers = {
   findTimes: function(e) {
     var deadlineMonitor = buildDeadlineMonitor(DEFAULT_DEADLINE_SECONDS);
     var settings = getSettingsForUser();
```

Comment:
is this settings var actually used?

Reply:
It is, but further down. Moved line closer to actual use.